### DISCUSSION TITLE
How can we use different languages ?

### DISCUSSION THREAD

**Discussion by ImrKhn03** [nlp-compromise/nlp_compromise#903] [General]
@spencermountain  Thank's for this package. I am intergrating my app to diffrent languages. I need to integrate this package with french language. Can you tell me how can i integrate this package with french language.

**Comment by spencermountain** [comment]
@ImrKhn03 please read the documentation at [fr-compromise](https://github.com/spencermountain/fr-compromise)

**Reply by ImrKhn03** [reply]
@spencermountain and for german?

**Reply by ImrKhn03** [reply]
thanks @spencermountain. I see you created different modules to support different languages.Can you please tell me how can i use other langueages with  [compromise](https://github.com/spencermountain/compromise) and not use other modules??

**Reply by spencermountain** [reply]
hi @ImrKhn03 compromise is built to work for english only, and to use other languages, you'll need to use the different modules.
There is documentation on each module. cheers

**Reply by ImrKhn03** [reply]
okay thanks @spencermountain.
